test/verify/check-selinux

```diff
@@ -77,6 +81,12 @@ class TestSelinux(MachineCase):
 
         self.login_and_go("/selinux/setroubleshoot")
 
+        # Starting Cockpit might produce some alerts (e.g. from
+        # starting rhsm.service), so let's give them some time to
+        # appear before cleaning it all out.
+        b.wait_js_cond("ph_in_text('body', 'No SELinux alerts.') || ph_is_present('tbody .pf-c-table__toggle')")
+        time.sleep(30)
```

Comment:
The sleep here is annoying indeed, but I don't have a better idea either. We could at least restrict it to rhel-8-6/centos-8-stream with a "HACK:" comment prefix, so that at some point we'll clean it up again.

But why the extra wait_js_cond state? The "while" loop below already does that, and it shouldn't matter much if the test sleeps first and then does the UI query, or the other way around?

Reply:
The idea was that we wait for the page to finish connecting to setroubleshoot first, since that seems to take a long time as well. But since thesubsequent wait needs to be really long anyway, *shrug*